core/src/main/java/com/google/googlejavaformat/java/Main.java

```diff
@@ -187,6 +189,19 @@ private int formatFiles(CommandLineOptions parameters, JavaFormatterOptions opti
         outWriter.write(formatted);
       }
     }
+
+    // #846 Clean up any resources and threads created by the executorService.
+    final boolean completedShutdown =
+        MoreExecutors.shutdownAndAwaitTermination(executorService, 100, TimeUnit.MILLISECONDS);
```

Comment:
I'm thinking to use a slightly longer deadline here, maybe 5 seconds.

This is just an FYI--I can make the change, no need to upload another snapshot.